components/Input/Input.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Input</title>
+    <link rel="stylesheet" href="./Input.css" />
+  </head>
+  <body>
+    <section class="flex-center flex-column" style="padding: 1rem">
```

Comment:
Suggestion: Add utility classes for different padding and margin

Reply:
Yes ! I have done that ,these inline styles will be removed during documentation phase , added here just so that it'll look better in preview